web/themes/custom/bevan/sass/components/_layout.scss

```diff
@@ -8,11 +8,4 @@
     position: relative;
     right: 50%;
     width: 100vw;
-}
-
-@media only screen and (max-width : 768px) {
-	.contextual-region {
```

Comment:
Sorry I reviewed the wrong PR
> The contextual-region class is added when you're logged in as an admin to Drupal to regions so it can add those pop up edit buttons. Is there another class we can use? This won't fix the problem for regular users